public/emails/welcome_on_signup.txt

```diff
@@ -0,0 +1,12 @@
+{{Subject .Subject "Welcome to Grafana"}}
+
+Hi {{.Name}},
+
+Welcome! Ready to start building some beautiful metric and analytic dashboards?
+
+If you are new to Grafana please read the Getting Started guid than can be found on
```

Comment:
See earlier edit

Reply:
@oddlittlebird See earlier reply ;)

Reply:
Fixed typos but did not use Markdown syntax